docs/source/user_guide/pipelines.ipynb

```diff
@@ -77,18 +85,13 @@
     "        \"parameter_1\":['a', 'b', 'c']\n",
     "    }\n",
     "    \n",
-    "    def __init__(self, parameter_1, random_seed):\n",
-    "        transformer = ThirdPartyTransformer(parameter_1)\n",
+    "    def __init__(self, parameter_1=1, random_seed=0):\n",
     "        parameters = {\"parameter_1\": parameter_1}\n",
     "        super().__init__(parameters=parameters,\n",
-    "                         component_obj=transformer,\n",
+    "                         component_obj=None,\n",
```

Comment:
Ahaha previously it didn't use a real third party transformer but it never error-ed because the code wasn't run... now that's no longer the case 😓  I can update this to use a real third party transformer and then comment it as such!

Reply:
Actually, we have info on this on our components doc which is linked so I'll just remove this line so it's not a focus.